pkg/image/api/validation/validation_test.go

```diff
@@ -315,24 +315,6 @@ func TestValidateImageStreamMappingNotOK(t *testing.T) {
 			field.ErrorTypeRequired,
 			"image.metadata.name",
 		},
-		"invalid repository pull spec": {
```

Comment:
Instead of removing, could you change the test to check this is a valid reference? Also could you add another case that would trigger `ErrorTypeInvalid`?